ivy/functional/frontends/torch/linalg.py

```diff
@@ -273,6 +273,35 @@ def solve(A, B, *, left=True, out=None):
     return ivy.solve(A, B, out=out)
 
 
+@to_ivy_arrays_and_back
+@with_supported_dtypes(
+    {"2.0.1 and below": ("float32", "float64", "complex32", "complex64")}, "torch"
+)
+def solve_triangular(A, b, upper=False):
```

Comment:
The function signature here is wrong, it should be `(A, B, *, upper, left=True, unitriangular=False, out=None)`
https://pytorch.org/docs/stable/generated/torch.linalg.solve_triangular.html